frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/PreviewPane/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { match } from "ts-pattern";
+
+type Theme = "light" | "dark";
+export function getCheckerBoardDataUri(theme: Theme) {
+  const [color1, color2] = match(theme)
+    .returnType<[string, string]>()
+    .with("light", () => ["#ededed", "transparent"])
+    .with("dark", () => ["#000000", "#323232"])
```

Comment:
See the [design here](https://www.figma.com/design/1YcsFf5yYdFZTguwoKVdkj/Re-design-the-transparent-background-setting-in-interactive-embedds?t=LfVJ5wqXyjEo2OCd-0).

I actually eye-balled the light-theme checkerboard since the one in the design is too dark, and the previous one is too pale. I chose ones in between.